app.rb

```diff
@@ -0,0 +1,100 @@
+# frozen_string_literal: true
+
+require 'sinatra'
+require 'json'
+set :erb, escape_html: true
+
+SAVE_FILE = 'memos.json'
+
+get '/' do
+  @memos = memos || []
+  erb :index if @memos
+end
+
+get '/edit/:id' do
+  @memo = get_memo(params[:id])
+  erb :edit if @memo
+end
+
+get '/new' do
+  erb :new
+end
+
+get '/show/:id' do
+  @memo = get_memo(params[:id])
+  erb :show if @memo
+end
+
+def memos
+  read_json[:memos]
+end
+
+def get_memo(memo_id)
+  memos[memo_id.to_sym]
+end
+
+def add_memo(memo_content)
+  memo_json = read_json
+  value = memo_json[:autoincrement].to_i + 1
```

Comment:
```suggestion
  new_id = memo_json[:autoincrement].to_i + 1
```

みたいな名前にした方がコードの意図がよりわかりやすくなりそうです

Reply:
修正します！

コミットID： 34a0d710b7f99d03ed2c3561290b77ed21d50c8b